src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -473,7 +473,8 @@ int sgn(T val)
 			// A new 'ArrayList' class is used to avoid an extra copy using List<T>.ToArray() here.
 			using (Stream stream = bitmap.PixelBuffer.AsStream())
 			{
-				_ = stream.WriteAsync(bgraPixelData.Array, 0, bgraPixelData.Count);
+				stream.Write(bgraPixelData.Array, 0, bgraPixelData.Count);
```

Comment:
Other platforms support this async. I am worried about performance changing this to sync code. I think it is better to fix the async support in Uno honestly.

Reply:
I can't really comment on the support of asynchronous features in Uno. But, testing has revealed that this was necessary for the spectrum to work on wasm (note to self: revalidate this).

Reply:
When I debugged this on macOS a long while ago I ran across the async issue which spanned a number of classes. When the async code completed, the image and brush was never invalidated and updated. The code to handle this simply doesn't exist on several platforms (Android/iOS seems to work though). This created an issue for other things such as drag/drop of images as well which is async by design.

Reply:
I checked again the code. All calls of `CreateBitmapFromPixelData()` I reviewed are already within async code blocks. This means `WriteAsync` here is redundant and using `Write` won't affect performance.